### ISSUE TITLE
Product Search (editor): missing label for <textarea> elements

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#1682]
![imatge](https://user-images.githubusercontent.com/3616980/73442110-7bb51400-4354-11ea-8e08-034ac6448ffd.png)

`<textarea>` elements are used to enter the label and placeholder, but they are missing a label, which is required for accessibility reasons.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea
